src/pages/documentation/release-notes/index.md

```diff
@@ -4,8 +4,188 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 8, 2023
+
+### iOS Mobile Core 4.x and compatible extensions
+
+New major version of the Mobile Core SDK for iOS has been released along with updates to other extensions. The current release includes the following changes:
+
+* Updated the minimum supported version to iOS 11.0 and tvOS 11.0.
```

Comment:
Since not all of these extensions support tvOS, it might be good to make a note of that here so that customers don't get the wrong idea. Maybe something like:

`...and tvOS 11.0 (for extensions which support tvOS)`